### DISCUSSION TITLE
Stability?

### DISCUSSION THREAD

**Discussion by unhannes** [Boulder-Investment-Technologies/lppls#76] [Q&A]
Everytime I generate the lppls model and compute the nested fit for the positive and negative bubble indicators, I get very different results.

Is this expected ? Is this due to some non-deterministic / probabilistic calculations? If so, is there a seed I can set somewhere?

Thanks in advance for any insights!

**Comment by Joshwani** [comment]
Hi @unhannes,
Yes, you're correct. The solver is not guaranteed to find the global minimum. Shu and Zhu (2019) propose CMA-ES for identifying the best estimation of the three non-linear parameters (tc, m, w). From my experience, this approach more consistently arrives at the same estimation. Unfortunately, it takes much more time to compute. 
Best,
Josh
